lib/ansible/modules/cloud/azure/azure_rm_policysetdefinition_facts.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Fan Qiu, (fanqiu@microsoft.com)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_policysetdefinition_facts
+version_added: "2.9"
+short_description: Gather Info of Policy Set Definition
+description:
+    - Gather information of a specific policy set definition or all policy set definitions
+
+options:
+    name:
+        description:
+            - Name of the policy set definition
+    management_group:
+        description:
+            - The ID of the management group.
+            - This parameter is mutually exclusive with 'built_in'.
+    built_in:
+        description:
+            - To gather all built-in policy set definitions when C(built_in) is set to True
+            - This parameter is mutually exclusive with 'management_group'.
+        type: bool
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Fan Qiu (@MyronFanQiu)"
+
+'''
+
+EXAMPLES = '''
+- name: List all policy set definitions
```

Comment:
in current tenant?

Reply:
in current subscription. I will add this information to adjust the name.